### PR TITLE
feat(java branching): added ansible tile and shorter screenshot in intro

### PR DESCRIPTION
**Pull request by akristen**
- Ansible logo
- Is this image still too long?

### CODE REVIEW

File: src/install/config/java.yaml [hunk 1]
```diff
@@ -72,6 +72,8 @@ appInfo:
       - value: 'aws'
         displayName: 'AWS Lambda'
         logo: 'awsLambda'
+      - value: 'ansible'
```
Comment (akristen) [inline src/install/config/java.yaml:75]: something is missing here*** AWS lambda immediately shortens and keeps the simple 2-step structure (a la VM). ask shawn about maneuvering this around

Comment (paperclypse) [reply src/install/config/java.yaml:75]: I just noticed this comment @akristen. Still need help with this?

Comment (akristen) [reply src/install/config/java.yaml:75]: No! We talked about it. I need to add value: ansible where the value: aws appears. :) Thanks for checking in!

**Review by paperclypse — APPROVED** [review]
Looks great!

**Pull request merged**